guide/src/class/protocols.md

```diff
@@ -424,6 +424,8 @@ cleared, as every cycle must contain at least one mutable reference.
   - `__traverse__(<self>, pyo3::class::gc::PyVisit<'_>) -> Result<(), pyo3::class::gc::PyTraverseError>`
   - `__clear__(<self>) -> ()`
 
+> Note: `__traverse__` does not work with [`#[pyo3(deprecated(...))]`](../function.md#deprecated) attribute.
+
```

Comment:
Does this (or using it on classattr) raise a compile error? Or does it just do nothing?

Reply:
Ah it currently does nothing. I should have made a compile error for that!